client/src/Components/Empty/index.jsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Empty } from 'antd';
 
-function EmptyCard() {
-  return <Empty description={false} />;
+function EmptyCard({description}) {
+  return <Empty description={
+    <span>
+      {description}
+    </span>
+  } />;
```

Comment:
Why did you use the `span` ?
```suggestion
  return <Empty description={description} />;
```